ROADMAP.md

```diff
@@ -0,0 +1,155 @@
+		- [1. Benchmark transfer performance in CI](#1-benchmark-transfer-performance-in-ci)
+		- [2. Add introspection to js-libp2p](#2-add-introspection-to-js-libp2p)
+- [libp2p Project Roadmap](#libp2p-project-roadmap)
+
+## About the Roadmap
+
+### Vision
+The tenets of the libp2p team are:
+- Security
+  - We strive to ensure that libp2p is reliable & secure for users & applications. We seek to minimize vulnerabilities and act quickly to address issues.
+- Stability
+  - We do not break deployed features and maintain cross version compatibility across libp2p releases.
+- Performance
+  - We aim to make libp2p as performant as possible and strive to ensure comparable performance to widely used protocols on the Internet. We avoid regressions and performance degradations in new releases.
+
+### Sections
+This document consists of two sections: [Milestones](#️-milestones) and the [Appendix](#-appendix)
+
+[Milestones](#️-milestones) is our best educated guess (not a hard commitment) around when we plan to ship the key features.
+Where possible projects are broken down into discrete sub-projects e.g. project "A" may contain two sub-projects: A.1 and A.2
+
+A project is signified as "complete" once all of it's sub-projects are shipped.
+
+The [Appendix](#-appendix) section describes a project's high-level motivation, goals, and lists sub-projects.
+
+Each Appendix header is linked to a GitHub Epic. Latest information on progress can be found in the Epics and child issues.
+
+### Done criteria
+The "Definition of Done" for projects/sub-projects that involve writing new protocols/ modify existing ones usually consist of the following:
+- If a specification change is required:
+    - [ ] Spec is merged and classified as "Candidate Recommendation"
+    - [ ] (by virtue of the above) At least one major reference implementation exists
+- [ ] A well established testing criteria is met (defined at the outset of the project including but not limited to testing via Testground, compatibility tests with other implementations in the Release process, etc.)
+- [ ] Public documentation (on docs.libp2p.io) exists
+
+Supporting projects (such as testing or benchmarking) may have different criteria.
+
+### Benchmarking and Testing
+As mentioned in our [vision](#vision), performance and stability are core libp2p tenets. Rigorous benchmarking and testing help us uphold them. Related projects are listed in the [libp2p/test-plans roadmap](https://github.com/libp2p/test-plans/pull/44) and the [testground/testground roadmap](https://github.com/testground/testground/pull/1484). Our major priorities in Q4’22 and Q1’23 are:
+- [interoperability testing](https://github.com/libp2p/test-plans/issues/53) (across implementations & versions and between transports, muxers, & security protocols)
+- [add a browser environment test harness to support testing browser features](https://github.com/testground/testground/issues/1386)
```

Comment:
I assume we'll basically have it from this but don't we also need the capability of having a nodejs testground environment?

Reply:
Yeah, it should be addressed by adding nix-builders to Testground. Added a line below